Scripts/Services/Pet Training/SpecialAbility.cs

```diff
@@ -1420,14 +1420,12 @@ public class StealLife : SpecialAbility
     {
         public override bool TriggerOnDoMeleeDamage => true;
         public override int ManaCost => 30;
+        public override TimeSpan CooldownDuration => TimeSpan.FromSeconds(Utility.Random(20, 40));
 
         public override void DoEffects(BaseCreature creature, Mobile defender, ref int damage)
         {
-            defender.FixedParticles(0x374A, 1, 15, 5054, 23, 7, EffectLayer.Head);
-            defender.PlaySound(0x1F9);
-
-            var dam = AOS.Damage(defender, creature, 45, 0, 0, 0, 0, 100);
-            creature.Hits = Math.Min(creature.HitsMax, dam);
+            var dam = AOS.Damage(defender, creature, 22, 0, 0, 0, 0, 100);
```

Comment:
according to this link: https://www.uo-cah.com/foundations/pet-abilities#lifeleech

The initial damage is 45.

Reply:
I checked in OSI. Initial damage is 22.

Reply:
Ok, but after resists?  This is energy damage, not direct damage.